build/helm/edgemesh/values.yaml

```diff
@@ -2,7 +2,7 @@ agent:
   image: kubeedge/edgemesh-agent:latest
   affinity: {}
   nodeSelector: {}
-  tolerations: {}
+  tolerations: []
```

Comment:
If you modified helm charts, then you must regenerate edgemesh.tgz with `helm package xxx` command.

Reply:
done.